apps/pwabuilder/src/script/components/android-form.ts

```diff
@@ -180,11 +180,8 @@ export class AndroidForm extends AppPackageFormBase {
 
   isMetaQuestChanged(checked: boolean) {
     this.packageOptions.isMetaQuest = checked;
-    if (checked) {
-      this.packageOptions.minSdkVersion = 23;
-    } else {
-      delete this.packageOptions.minSdkVersion;
-    }
+    // Always keep minSdkVersion at 23 as Google Play Console no longer accepts lower versions
+    this.packageOptions.minSdkVersion = 23;
```

Comment:
Given that emptyAndroidPackageOptions already includes minSdkVersion, this code could be safely removed.

Reply:
Thanks for pointing that out! If you think this can be safely removed, feel free to go ahead and make the change.